v3/internal/assetserver/webview/responsewriter_windows.go

```diff
@@ -68,7 +69,7 @@ func (rw *responseWriter) Finish() error {
 	if code == http.StatusNotModified {
 		// WebView2 has problems when a request returns a 304 status code and the WebView2 is going to hang for other
```

Comment:
This was committed in September 2023. I've scoured the Webview2 feedback site and can't find anything about this status code 🤔